wrapper/src/test/java/integration/container/TestDriverProvider.java

```diff
@@ -165,6 +165,10 @@ public void beforeEach(ExtensionContext context) throws Exception {
                       try {
                         instanceIDs = auroraUtil.getAuroraInstanceIds();
                       } catch (SQLException ex) {
+                        if ("28P01".equals(ex.getSQLState())) {
```

Comment:
NIT: can we do a global static variable with the name of the error code? like POSTGRES_AUTH_ERROR_CODE